shared/wallets/asset/container.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import {Assets} from '.'
+import {connect, type TypedState, type Dispatch} from '../../util/container'
+import {getAssets, getSelectedAccount} from '../../constants/wallets'
+
+const mapStateToProps = (state: TypedState) => {
+  const selectedAccount = getSelectedAccount(state)
+  const assets = getAssets(state, selectedAccount)
```

Comment:
we could make these helpers just default to the selected account if you don't pass one in. likely 95% of the time this is what we'll want